infra/app/appserviceplan.bicep

```diff
@@ -0,0 +1,20 @@
+param name string
```

Comment:
Please use what we have in our templates under /infra/core and put those files in your project under infra core and use /infra/app for app specific files.

Reply:
restructured the files by following the same pattern as azd-dev.